salt/utils/reactor.py

```diff
@@ -346,7 +346,7 @@ def caller(self, fun, *args, **kwargs):
         Wrap Caller to enable executing :ref:`caller modules <all-salt.caller>`
         '''
         log.debug("in caller with fun {0} args {1} kwargs {2}".format(fun, args, kwargs))
-        args = kwargs.get('args', [])
```

Comment:
I don't think 'args' is ever going to be in kwargs. format_call seems never to populate it that way. Possibly iff it's using the state name as given to it by the minion reactor? maybe?

This would certainly work in my case though. Changing.